### ISSUE TITLE
Eonasdan's bootstrap datetimepicke

### ISSUE THREAD

**Issue by Mo3ta9em93** [xsoh/moment-hijri#23]
Thanks for this git
Please take a look at Eonasdan's bootstrap datetimepicker[https://github.com/Eonasdan/bootstrap-datetimepicker](url)
it uses moment with locales
is there any way i can implement your script with it
the page freezes when i load moment-hijri
thank you very much

**Comment by HakamFostok** [comment]
You are completely right. the page is completely freezing with the bootstrap datetimepicker.
This library is not working with bootstrap datetimepicker at all.
